source/about/release-policy.md

```diff
@@ -45,7 +45,25 @@ To install extended support releases, follow our [install](/guides/deployment) o
 We highly recommend working with your Mattermost Account Team to plan for a migration to our new plans, and to access the latest features such as persistent notifications, advanced compliance features, call recordings, and more.
 ```
 
-![Timeline-based chart showing the lifecycle of Mattermost Extended Support Releases and feature releases from last year and this year.](../images/ESR_graphic2.png)
+```{mermaid}
+
+gantt
+    dateFormat  YYYY-MM-DD
+    axisFormat  %b
+    
+    section 2024
+    v9.4                   :done,    2024-01-16, 2024-04-15
+    v9.5 Extended Support  :crit,    2024-02-16, 2024-11-15
+    v9.6                   :done,  2024-03-16, 2024-06-15
+    v9.7                   :done,  2024-04-16, 2024-07-15
+    v9.8                   :active,  2024-05-15, 2024-08-15
+    v9.9                   :active,  2024-06-16, 2024-09-15
+    v9.10                  :active,  2024-07-16, 2024-10-15
+    v9.11 Extended Support :crit,    2024-08-16, 2025-05-15
+
+    section 2025
+    v9.11 Extended Support :crit,    2024-08-16, 2025-05-15
```

Comment:
v9.11 is shown twice on the graph.

Otherwise all looks great!

Reply:
I suspect we can simply delete the duplicate line. @harshilsharma63 - can you confirm, please?

Reply:
Seems like, but thats not something I wrote. I tried it and it looks file, but note that the year sections for 2024 and 2025 are horizontal, not vertical as they used to be before. I caouldn't find a  way to make them vertical in Mermaid, so its just what it is.

But yeah, the duplicate line can be removed.

Reply:
I too couldn't find a way to make the year sections match the original image. Thanks for confirming that we can remove that duplicate line, @harshilsharma63!

Reply:
@amyblais - I tried removing that duplicate line, and the result is that the year 2025 disappears from the chart. I propose that we replace the duplicate line with 1 or more upcoming ESR releases instead, if they're known.

Reply:
I've updated the versions now, but I see a few issues in the [latest preview](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7305/about/release-policy.html):
1. ``v10.4`` version is missing from the graph for some reason.
2. The v10.1 - v10.4 should be purple colour.

Reply:
I've updated the versions now - I see one issue in the [latest preview](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7305/about/release-policy.html) where the v10.1 -> v10.4 versions should be purple, and the v10.5 ESR version should be pink, and I wasn't sure how to fix it.

Reply:
@harshilsharma63 - Can we get your help implementing Amy's feedback, please?

Reply:
@amyblais @cwarnermm those releases were appearing because they were specified as `:crit`. Anytime you want a release to show as purple, mark them as `:done`, `:active` or `:upcoming` and for pink, mark them as `:crit`.

For example, changing

```
    v10.0                  :crit,    2024-09-16, 2024-12-15
```

to 

```
    v10.0                  :upcoming,    2024-09-16, 2024-12-15
```

resolved the colors from pink to purple.

Reply:
![Screenshot 2024-08-14 at 10 10 17 AM](https://github.com/user-attachments/assets/6136357e-1a51-4767-8e70-9d3da75ca19e)

Looks correct now

Reply:
Thank you for the guidance, @harshilsharma63!

Reply:
Thank you!